handler/status.go

```diff
@@ -22,11 +22,13 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
 	kingpin "gopkg.in/alecthomas/kingpin.v2"
 
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
-	"github.com/prometheus/common/log"
+
```

Comment:
So yeah, the convention hasn't even been followed precisely here. But you would make me very happy if you fixed it while adding the now imports. It should be: One block for standard imports, one for unnamed imports from other repos, one for named imports from other repos, one for unnamed imports from the same repo, and finally one for named imports from the same repo.

Thus, the now go-kit imports should be in this block. the kingpin import should be separate below, and again separate and at the bottom should be the pushgateway/storage import.

Reply:
In different news: I'm not even sure we need to name the `kingpin` import (may be done as in `main.go`).